vast/vastelement/inline.go

```diff
@@ -21,7 +21,7 @@ type InLine struct {
 
 // Validate methods validate the Inline element according to the VAST.
 // AdSystem, AdTitle, Impression, Creatives are required.
-func (inline *InLine) Validate(version Version) error {
+func (inline *InLine) Validate(version Version, needFilterURI bool) error {
```

Comment:
I remember last time someone said we can pass a func here for user to customize the checking?